src/functions/schema.ts

```diff
@@ -33,26 +34,40 @@ const logger = {
   error: console.error,
 };
 
-const ajv = new AJV({
-  meta: false,
-  schemaId: 'auto',
-  jsonPointers: true,
-  unknownFormats: 'ignore',
-  logger,
-});
-ajv.addMetaSchema(jsonSpecv4);
-ajv.addMetaSchema(jsonSpecv6);
-ajv.addMetaSchema(jsonSpecv7);
-// @ts-ignore
-ajv._opts.defaultMeta = jsonSpecv4.id;
-// @ts-ignore
-ajv._refs['http://json-schema.org/schema'] = 'http://json-schema.org/draft-04/schema';
-
-ajv.addFormat('int32', { type: 'number', validate: oasFormatValidator.int32 });
-ajv.addFormat('int64', { type: 'number', validate: oasFormatValidator.int64 });
-ajv.addFormat('float', { type: 'number', validate: oasFormatValidator.float });
-ajv.addFormat('double', { type: 'number', validate: oasFormatValidator.double });
-ajv.addFormat('byte', { type: 'string', validate: oasFormatValidator.byte });
+const ajvInstances = {};
+
+function getAjv(oasVersion: number = 0): AJV.Ajv {
```

Comment:
```suggestion
function getAjv(oasVersion: Optional<number | null>): AJV.Ajv {
```
Perhaps let's make it nullable & optional.

Reply:
`Optional` can be imported from `@stoplight/types`.

Reply:
I'll try my best with my limited TS knowledge to make the changes tomorrow.